smoke-test/tests/privileges/test_privileges.py

```diff
@@ -450,4 +450,64 @@ def test_privilege_to_create_and_manage_policies():
 
 
     # Ensure that user can't create a policy after privilege is removed by admin
-    _ensure_cant_perform_action(user_session, create_policy,"createPolicy")
\ No newline at end of file
+    _ensure_cant_perform_action(user_session, create_policy,"createPolicy")
+
+
+@pytest.mark.dependency(depends=["test_healthchecks"])
+def test_privilege_from_group_role_can_create_and_manage_tokens():
+
+    (admin_user, admin_pass) = get_admin_credentials()
+    admin_session = login_as(admin_user, admin_pass)
+    user_session = login_as("user", "user")
+    secret_urn = "urn:li:dataHubSecret:TestSecretName"
+
+    # Verify new user can't create secrets
+    create_secret = {
+        "query": """mutation createSecret($input: CreateSecretInput!) {\n
+            createSecret(input: $input)\n}""",
+        "variables": {
+            "input":{
+                "name":"TestSecretName",
+                "value":"Test Secret Value",
+                "description":"Test Secret Description"
+            }
+        },
+    }
+    _ensure_cant_perform_action(user_session, create_secret,"createSecret")
```

Comment:
Please update the smoke test to use listSecrets endpoint after this attempt and ensure secret has not been created. Given the second check I am not sure how reliable this method is.